src/index.html

```diff
@@ -46,6 +46,9 @@ <h2 class="sidebar-header-name" id="sidebarName">Name Name</h2>
           <div class="card-today-hydration">
             <p id="hydrationToday"></p>
           </div>
+          <p>Max amount is 128oz</p>
+          <input type="text" class="new-hydration-input" placeholder="Number of ounces" onkeyup="value=value.replace(/[^\d]/g,'')" onkeyup=>
```

Comment:
I think you said you have an explanation to share with us about regex. I'm excited to see it!